src/components/CreateListButton.js

```diff
@@ -0,0 +1,20 @@
+import { getToken } from '@the-collab-lab/shopping-list-utils';
+import { NavLink } from 'react-router-dom';
+
+function CreateListButton({ setToken }) {
+  function createTokenAndSaveToLocalStorage() {
+    const token = window.localStorage.setItem(
+      'userToken',
+      JSON.stringify(getToken()),
```

Comment:
I don't think you need `JSON.stringify`. `getToken()` returns a string - see [function here](https://github.com/the-collab-lab/shopping-list-utils/blob/cb03fd55aaddf9ee0fe40189a710483c548409a6/src/tokens/index.ts#L34).